<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>802</width>
    <height>344</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>840</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Current Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_lbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>320</width>
      <height>240</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(69, 209, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="img_bg_lbl">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>30</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ctrl_arrow_lbl">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(205, 190, 255);
border: 2px solid #222222;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Histogram</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Control Arrow key</string>
    </property>
   </widget>
   <widget class="QLabel" name="img_his_lbl">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>190</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 175, 177);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>0</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Background Image</string>
    </property>
   </widget>
   <widget class="QLabel" name="ctrl_mouse_lbl">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>30</y>
      <width>160</width>
      <height>120</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(255, 255, 255);
background-color: rgb(103, 248, 255);
border: 2px solid #222222;
border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>0</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>Control mouse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="change_bg_btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #222222;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Change BG</string>
    </property>
   </widget>
   <widget class="QLabel" name="print_percent_lbl">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>90</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 217, 24);
border: 2px solid #222222;
border-radius: 10px;
font: 75 15pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>%</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="browser_btn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #222222;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Browser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_program_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #222222;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Start Program</string>
    </property>
   </widget>
   <widget class="QPushButton" name="start_process_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>100</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 2px solid #222222;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string>Start Process</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>133</y>
      <width>101</width>
      <height>141</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QCheckBox" name="show_per_cb">
       <property name="text">
        <string>Show_Per</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="sub_bg_cb">
       <property name="text">
        <string>Sub_BG</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="equal_his_cb">
       <property name="text">
        <string>Equal_His</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="draw_his_cb">
       <property name="text">
        <string>Draw_His</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ctrl_mouse_cb">
       <property name="text">
        <string>Ctrl_Mouse</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ctrl_arrow_cb">
       <property name="text">
        <string>Ctrl_Arrow</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="hand_lbl">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>280</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 14pt &quot;.VnBlackH&quot;;
color: rgb(255, 14, 14);</string>
    </property>
    <property name="text">
     <string>---o---</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="print_num_finger_lbl">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(43, 252, 255);
border: 2px solid #222222;
border-radius: 10px;
font: 75 15pt &quot;Times New Roman&quot;;</string>
    </property>
    <property name="text">
     <string>n</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>layoutWidget</zorder>
   <zorder>label_5</zorder>
   <zorder>img_lbl</zorder>
   <zorder>img_bg_lbl</zorder>
   <zorder>ctrl_arrow_lbl</zorder>
   <zorder>label_2</zorder>
   <zorder>label_4</zorder>
   <zorder>img_his_lbl</zorder>
   <zorder>label</zorder>
   <zorder>ctrl_mouse_lbl</zorder>
   <zorder>label_3</zorder>
   <zorder>print_percent_lbl</zorder>
   <zorder>browser_btn</zorder>
   <zorder>start_program_btn</zorder>
   <zorder>start_process_btn</zorder>
   <zorder>change_bg_btn</zorder>
   <zorder>hand_lbl</zorder>
   <zorder>print_num_finger_lbl</zorder>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
